modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -108,7 +108,9 @@ After approving on the *Install Plan* page, the subscription upgrade status move
 
 .. If you selected an *Automatic* approval strategy, the upgrade status should resolve to *Up to date* without intervention.
 
-. After the upgrade status of the subscription is *Up to date*, select *Operators → Installed Operators* to verify that the cluster service version (CSV) of the installed Operator eventually shows up. The *Status* should ultimately resolve to *InstallSucceeded* in the relevant namespace.
+.Verification
+
+* After the upgrade status of the subscription is *Up to date*, select *Operators → Installed Operators* to verify that the cluster service version (CSV) of the installed Operator eventually shows up. The *Status* should ultimately resolve to *InstallSucceeded* in the relevant namespace.
```

Comment:
Seems like it only shows the `Succeeded`, not the `InstallSucceeded`.
<img width="1684" alt="image" src="https://github.com/user-attachments/assets/4104c5de-7b35-4321-a5c7-8886f0e5c816">